library/src/main/java/com/getstream/sdk/chat/utils/MessageInputClient.java

```diff
@@ -143,13 +143,19 @@ private void configSelectAttachView(boolean isMedia, List<Attachment> editAttach
         if (isMedia) {
             List<Attachment> attachments = Utils.getAllShownImagesPath(context);
             ((Activity) context).runOnUiThread(() -> {
-                mediaAttachmentAdapter = new MediaAttachmentAdapter(context, attachments, position -> {
-                    Attachment attachment = attachments.get(position);
-                    attachment.config.setSelected(!attachment.config.isSelected());
-                    mediaAttachmentAdapter.notifyItemChanged(position);
-                    updateComposerViewBySelectedMedia(attachments, attachment);
-                });
-                binding.rvMedia.setAdapter(mediaAttachmentAdapter);
+                if (!attachments.isEmpty()){
+                    mediaAttachmentAdapter = new MediaAttachmentAdapter(context, attachments, position -> {
+                        Attachment attachment = attachments.get(position);
+                        attachment.config.setSelected(!attachment.config.isSelected());
+                        mediaAttachmentAdapter.notifyItemChanged(position);
+                        updateComposerViewBySelectedMedia(attachments, attachment);
+                    });
+                    binding.rvMedia.setAdapter(mediaAttachmentAdapter);
+                }else{
+                    Utils.showMessage(context, "There is no photo or video.");
```

Comment:
Could you also put strings into `strings.xml`? Sooner or later we will need to align texts with other mobile platforms and provide way to change/translate texts for developers. And that's just easier to do from the beginning, rather that later trying to extract all strings.

Reply:
added code to check file existing when getting files from gallery.